content/general/changelog.md

```diff
@@ -6,7 +6,17 @@ weight: 4
 
 Below you can find a full list of changes to VU, corresponding to every VU update, with the newest update at the top. Updates that have the `ONLY IN DEV` tag next to them are currently only available in the `dev` branch.
 
-{{% changeset 19412 "11/09/2022" true %}}
+{{% changeset 19549 "08/10/2022" true %}}
+- VeniceEXT was updated to `1.11.0` ([changelog](/vext/changelog/#1_11_0)).
+- Add support for `AutoTeamEntityData.rotateTeamOnNewRound`. 
```

Comment:
It's a bit unclear as to what this means and how it affects squad / team management.